### DISCUSSION TITLE
What's the difference between Next w/ SSR and Blitz's Data Layer offering?

### DISCUSSION THREAD

**Discussion by oasaleh** [blitz-js/blitz#4255] [General]
With Next.js 14 and server components, how will this phrase from Blitz's website hold?

<blockquote>
2. Typesafe Data Layer
Blitz popularized RPC as an alterative to REST or GraphQL APIs. With Blitz, you can import your server code directly into the frontend so you don't need to build an API and do data fetching from the frontend. At build time, Blitz automatically inserts an RPC API call that runs the server code on the server. Essentially, Blitz abstracts your API into a compile step.
</blockquote>

Built-in DB integration and auth are big. But I'm only interested in the above point. I'm trying to understand the advantage/disadvantage of using Blitz over newest Next with SSR.

**Comment by cfragkos** [comment]
Even though late I will answer because I have been thinking a lot about this. Especially if you have a production app, there are still benefits for blitz. For starters you can use both. So use blitz rpc instead of trpc for example, but you can call the underlying function server side. A reason to do this is if for example you need your methods to be externally consumed at the same time which with server actions won't happen and native Route Handlers are not better than blitz rpc at least  in their DX. 
Only thing that worries me is blitz stops doing updates in the near future. 

Blitz still has a lot to offer but for some reason it is not being pushed by the team and seems to be slowly dying

**Reply by rafaelm7o** [reply]
Yep, it offers a lot 
But after planning the 2.0 version, it looks like it is in maintenance mode only
The boom got quiet after early 2022
X/Twitter and the Discord server are neglected
